README.md

```diff
@@ -1,22 +1,35 @@
 # Internet-in-a-Box (IIAB)
 
-Welcome to the Git repository of the Internet-in-a-Box (IIAB) project. This is a community-based project developed and supported by volunteers from around the world. The Internet-in-a-Box (IIAB) is small, inexpensive device which provides essential Internet resources (like Wikipedia, OpenStreetMap, Khan Academy and others) without any Internet connection. It provides a local content server of the world’s Free Knowledge.
-
-The older version of this repository is at [github.com/xsce](http://github.com/xsce). We are using [Ansible](https://www.ansible.com) as the underlying technology to install, deploy, configure and manage the various software components.
+Welcome to the Git repository for Internet-in-a-Box (IIAB).  This community-based project is enabled by professional volunteers around the world.  Internet-in-a-Box (IIAB) is a small, inexpensive device which provides essential Internet resources (like Wikipedia, OpenStreetMap, Khan Academy and others) without any Internet connection.
 
+You are encouraged to customize this local content hotspot or server with the best of the world’s Free Knowledge — to suit your region, your school, your medical establishment, your library — or for your very own family.
 
 ## Installation
 
-Please read the [Installation](https://github.com/iiab/iiab/wiki/IIAB-Installation) wiki page for details about installation of Internet-in-a-Box (IIAB).
+Please read the [Installation](https://github.com/iiab/iiab/wiki/IIAB-Installation) wiki page for details on installing of Internet-in-a-Box (IIAB).
+
+Or, to dive right in, our "1-line installer" is available off of http://download.iiab.io — it's generally best to click on the highest version number.
 
-## Built With
+Please see "What are the best places for community support?" at http://FAQ.IIAB.IO which many answers to help you along the way, in designing the digital knowledge hotspot most suitable for your own community.
 
-* [Ansible](http://www.ansible.com)
+#### Ansible IT Automation
+
+FYI we are using [Ansible](https://www.ansible.com) as the underlying technology to install, deploy, configure and manage the various software components.
 
 ## Contributing
 
-Please read our [Contributors Guide](https://github.com/iiab/iiab/wiki/IIAB-Contributors-Guide) to learn more about contributing to Internet-in-a-Box (IIAB) repository.
+We greatly welcome contributions from educators, librarians *and* IT/UX/QA people of all kind!
+
+Please see "How can I help?" at http://FAQ.IIAB.IO
+
+Check out 2017's [Contributors Guide](https://github.com/iiab/iiab/wiki/IIAB-Contributors-Guide) to learn more about contributing directly to Internet-in-a-Box (IIAB) software and its open community architecture for education.
+
+*Thank you for helping us enable offline access to the Internet's open knowledge jewels, as well as "Sneakernet-of-Alexandria" distribution of local/indigenous content, where mass media channels are not serving grassroots voices.*
```

Comment:
Why is this paragraph italicized?

Reply:
Catalyzing grassroots education is important motivating force here, that mobilizes many to contribute.

(Or we can un-italicize later if this emphasis belongs elsewhere?)